stdlib/bdb.pyi

```diff
@@ -2,15 +2,15 @@ import sys
 from _typeshed import ExcInfo, TraceFunction, Unused
 from collections.abc import Callable, Iterable, Mapping
 from types import CodeType, FrameType, TracebackType
-from typing import IO, Any, Literal, SupportsInt, TypeVar
+from typing import IO, Any, Final, SupportsInt, TypeVar
 from typing_extensions import ParamSpec
 
 __all__ = ["BdbQuit", "Bdb", "Breakpoint"]
 
 _T = TypeVar("_T")
 _P = ParamSpec("_P")
 
-GENERATOR_AND_COROUTINE_FLAGS: Literal[672]
+GENERATOR_AND_COROUTINE_FLAGS: Final = 672
```

Comment:
Hmm. The definition of this constant at runtime is

```py
from inspect import CO_GENERATOR, CO_COROUTINE, CO_ASYNC_GENERATOR

GENERATOR_AND_COROUTINE_FLAGS = CO_GENERATOR | CO_COROUTINE | CO_ASYNC_GENERATOR
```

And the `inspect` docs say:

> Note: The flags are specific to CPython, and may not be defined in other Python implementations. Furthermore, the flags are an implementation detail, and can be removed or deprecated in future Python releases. It’s recommended to use public APIs from the [inspect](https://docs.python.org/3/library/inspect.html#module-inspect) module for any introspection needs.

As such, I think it's correct to mark this constant as `Final`, but I think we probably shouldn't include the exact value in the stubs at all. I recommend changing this as follows:

```suggestion
# A union of code-object flags at runtime.
# The exact values of code-object flags are implementation details,
# so we don't include the value of this constant in the stubs.
GENERATOR_AND_COROUTINE_FLAGS: Final[int]
```